build.gradle

```diff
@@ -8,6 +8,12 @@ group = 'uk.gov.digital.ho.hocs'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
 
+configurations {
+	compileOnly {
```

Comment:
Is this a necessity?

Reply:
It came with the springboot initilizr when you add lombok, it works without it too.